source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -44,6 +47,9 @@ For more complete performance testing, consider using a combination of load-test
 
    Measure the network throughput of all nodes.
 
+   :mc-cmd:`mc support perf net` temporarily suspends S3 API calls during the test.
+   Incoming requests are queued until the test is complete or the command is cancelled.
```

Comment:
Alternative for you to consider to avoid all of the passives.
If you use it here, use it in both places.

```suggestion
   Incoming requests are held in a queue while the command runs.
   When the command completes or ends, queued requests process as normal.
```

Reply:
Derp 🤦🏻

Reply:
used variant of your suggestion 💯